### PR TITLE
[RW-3327][RISK=NO]/dataSet returning 500 : add null constraint 

### PR DESCRIPTION
**Pull request by NehaBroad**
Adding database script to add null constraint and also add default values as 0 to prePackagedConceptSet column

### CODE REVIEW

**Comment by jaycarlton** [comment]
Are there any other columns on the data_set table that are nullable but don't have a default value?

File: api/db/changelog/db.changelog-109-dataset-prepackaged-conceptsets-nullConstraint.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="nsaxena" id="db.changelog-109-dataset-prepackaged-conceptsets-nullConstraint">
+    <addNotNullConstraint columnDataType="tinyint"
+            columnName="prePackagedConceptSet"
```
Comment (jaycarlton) [inline api/db/changelog/db.changelog-109-dataset-prepackaged-conceptsets-nullConstraint.xml]: Should this column be renamed `pre_packaged_concept_set_id`?

Comment (NehaBroad) [reply api/db/changelog/db.changelog-109-dataset-prepackaged-conceptsets-nullConstraint.xml]: should we tackle that in another PR?

Comment (jaycarlton) [reply api/db/changelog/db.changelog-109-dataset-prepackaged-conceptsets-nullConstraint.xml]: I'd most likely do both in one PR, though you could make it two separate migrations. Not required, but maybe file a ticket for this and any other columns that don't match the convention.

**Pull request merged**